libultraship/libultraship/ImGuiImpl.cpp

```diff
@@ -72,6 +72,39 @@ namespace SohImGui {
         "None"
     };
 
+    const char* powers256[9] = {
+        "Vanilla (1x)",
+        "Double (2x)",
+        "Quadruple (4x)",
+        "Octuple (8x)",
+        "Hexadecuple (16x)",
+        "Duotrigintuple (32x)",
+        "Quattuorsexagintuple (64x)",
+        "Octoviginticentuple (128x)",
+        "Hexaquinquagintiducentuple (256x)"
+    };
+
+    const char* powers128[8] = {
+        "Vanilla (1x)",
+        "Double (2x)",
+        "Quadruple (4x)",
+        "Octuple (8x)",
+        "Hexadecuple (16x)",
+        "Duotrigintuple (32x)",
+        "Quattuorsexagintuple (64x)",
+        "Octoviginticentuple (128x)"
+    };
+
+    const char* powers64[7] = {
+        "Vanilla (1x)",
+        "Double (2x)",
+        "Quadruple (4x)",
+        "Octuple (8x)",
+        "Hexadecuple (16x)",
+        "Duotrigintuple (32x)",
+        "Quattuorsexagintuple (64x)"
+    };
```

Comment:
this feels like copypasta, is there a way to do this without manually making 3 arrays?

Reply:
I was thinking about assigning the other pointers to the first and seeing if it worked, but forgot to do it. That was a temporary implementation that I forgot to un-temporary-ify. Will try to do that now.